BridgeSDK/BridgeAPI/SBBAuthManager.h

```diff
@@ -125,18 +128,15 @@ extern  NSString * _Nonnull gSBBAppStudy;
  */
 - (nullable NSString *)usernameForAuthManager:(nullable id<SBBAuthManagerProtocol>)authManager __attribute__((deprecated("implement emailForAuthManager: instead")));
 
-
 /*!
- *  This delegate method should return the password for the user account last signed up for or signed in to,
- *  or nil if the user has never signed up or signed in on this device.
+ *  For backward compatibility only. This method will no longer be called.
  *
- *  @note This method is optional. If both this and emailForAuthManager: are provided by the delegate, the SDK can handle refreshing the session token automatically when 401 status codes are received from the Bridge API.
+ *  @deprecated Implement authManager:didGetSessionToken:forEmail:andPassword instead.
  *
  *  @param authManager The auth manager instance making the delegate request.
- *
- *  @return The password, or nil.
+ *  @param sessionToken The session token just obtained by the auth manager.
  */
-- (nullable NSString *)passwordForAuthManager:(nullable id<SBBAuthManagerProtocol>)authManager;
+- (void)authManager:(nullable id<SBBAuthManagerProtocol>)authManager didGetSessionToken:(nullable NSString *)sessionToken __attribute__((deprecated("implement authManager:didGetSessionToken:forEmail:andPassword instead")));
```

Comment:
yes? it's deprecated, like seriously don't-use-this-anymore deprecated, the other one with all the parameters is now required, and this was always the fallback so with the other one always implemented this would never get called anyway. the only reason it's still defined at all is so you get a deprecation message telling you to use the other one instead of just a build error.